src/components/generic/ContentVersionControls.tsx

```diff
@@ -0,0 +1,446 @@
+import * as React from "react";
+import { Box, Button, Divider, List, ListItem, ListItemIcon, ListItemText, Popover, Tooltip, Typography, WithStyles, withStyles } from "@material-ui/core";
+import { ReduxDispatch, ReduxState } from "app/store";
+import { connect, ConnectedProps } from "react-redux";
+import styles from "styles/generic/content-version-controls";
+import { addContentVersion, deleteContentVersion, selectContentVersion, updateContentVersion } from "features/content-version-slice";
+import { ApiRequestMetadata, ContentVersion, ErrorContextType } from "types";
+import strings from "localization/strings";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import AddIcon from "@material-ui/icons/AddCircle";
+import PlayCircleOutlineOutlinedIcon from '@material-ui/icons/PlayCircleOutlineOutlined';
+import { setApplication } from "features/application-slice";
+import { Application, ResourceType } from "generated/client";
+import Api from "api";
+import { toast } from "react-toastify";
+import { ErrorContext } from "components/containers/ErrorHandler";
+import AddContentVersionDialog from "./AddContentVersionDialog";
+import moment from "moment";
+
+/**
+ * Component properties
+ */
+interface Props extends ExternalProps { }
+
+/**
+ * Component state
+ */
+interface State {
+  popOverAnchor: HTMLElement | null;
+  addContentVersionDialogOpen: boolean;
+}
+
+/**
+ * Content version controls
+ */
+class ContentVersionControls extends React.Component<Props, State> {
+
+  static contextType: React.Context<ErrorContextType> = ErrorContext;
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      popOverAnchor: null,
+      addContentVersionDialogOpen: false
+    }
+  }
+
+  /**
+   * Component render
+   */
+  public render = () => {
+    const { classes, contentVersions } = this.props;
+
+    if (!contentVersions.length) {
+      return <div/>;
```

Comment:
```suggestion
      return null;
```

Reply:
This does matter since the other DOM element under the the same parent element as this uses its sibling, aka this element, to its positioning. If I remove this, I need to change the other elements styles too.